pkg/apiserver/rest/dualwriter_mode1_test.go

```diff
@@ -13,6 +14,20 @@ import (
 	"k8s.io/apiserver/pkg/apis/example"
 )
 
+// waitPendingActions is a workaround to allow consistent testing.
+//
+// FIXME: DualWriter interface to provide a graceful termination mechanism. The
```

Comment:
We should consider discussing a server lifecycle approach. The server should be able to signal when it has completed all of its background work. This can be done with a `Run` and a  `Shutdown`/`Stop`/etc. methods. Let's discuss this offline.

cc: @leonorfmartins